docs/components/hero.md

```diff
@@ -0,0 +1,30 @@
+#Hero
+
+Add the `HeroMixin` to your page model:
+```py
+from wagtailnhsukfrontend.mixins import (
+    ReviewDateMixin,
+)
+
+class MyPage(HeroMixin, Page):
+
+    ...
+
+    content_panels = Page.content_panels + [HeroMixin.hero_contents, ...]
+```
+
+The `HeroMixin` will add fields to your page model, allowing the Hero component to be edited in the
+contents panels of the wagtail cms.
+
+To include the Hero component on a page you will have to include the Hero template in your page template:
+'''django
+  {%include "wagtailnhsukfrontend/hero.html"%}
```

Comment:
Should be whitespace around django tags
`{% include "wagtailnhsukfrontend/hero.html" %}` instead of {%include "wagtailnhsukfrontend/hero.html"%}